mocked-api/handlers.ts

```diff
@@ -254,6 +254,73 @@ export const handlers = [
 
     return res(ctx.status(200), ctx.delay(apiDelay), ctx.json(newBridge));
   }),
+  // update a bridge
+  rest.put(`${apiUrl}/bridges/:bridgeId`, async (req, res, ctx) => {
+    const { bridgeId } = req.params;
+    const bridgeRequest: BridgeRequest = await req.json();
+    const { name, error_handler: updatedErrorHandler } = bridgeRequest;
+
+    const existingBridge = db.bridge.findFirst({
+      where: {
+        id: {
+          equals: bridgeId as string,
+        },
+      },
+    });
+
+    if (!existingBridge) {
+      return res(
+        ctx.status(404),
+        ctx.delay(apiDelay),
+        ctx.json({
+          kind: "ErrorsResponse",
+          items: [
+            {
+              ...error_not_found,
+              reason: `Bridge with id '${
+                bridgeId as string
+              }' for customer 'XXXXXXXX' does not exist`,
+            },
+          ],
+        })
+      );
+    }
+
+    const updatedBridge = db.bridge.update({
+      where: {
+        id: {
+          equals: bridgeId as string,
+        },
+      },
+      data: {
+        ...bridgeRequest,
+        error_handler: {
+          type: updatedErrorHandler?.type,
+          parameters: JSON.stringify(updatedErrorHandler?.parameters),
+        },
+      },
```

Comment:
When updating the bridge you also have to set new values for `status` and `modified_at`, similarly to what we did for processors.

https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/c0f582abcc12337580a8c15c933141372d9a6fb3/mocked-api/handlers.ts#L697-L705